code/processes/tickerlogreplay.q

```diff
@@ -27,7 +27,7 @@ clean:@[value;`clean;1b]                                                // clean
 sortcsv:@[value;`sortcsv;hsym first .proc.getconfigfile["sort.csv"]]    // location of  sort csv file
 compression:@[value;`compression;()];                                   // specify the compress level, empty list if no required
 partandmerge:@[value;`partandmerge;0b];                                 // setting to do a replay where the data is partitioned and then merged on disk
-tempdir:@[value;`tempdir;`:tempmergedir];                               // location to save data for partandmerge replay
+tempdir:hsym @[value;`tempdir;`:tempmergedir];                               // location to save data for partandmerge replay
```

Comment:
I don't think we need to do this here seeing as we're doing it below - but if we are leaving it here as well, need to re-align the comment